src/astra_driver.cpp

```diff
@@ -717,6 +720,7 @@ sensor_msgs::msg::CameraInfo::SharedPtr AstraDriver::getDepthCameraInfo(int widt
 
 void AstraDriver::readConfigFromParameterServer()
 {
+  depth_frame_id_ = std::string("openni_depth_optical_frame");
```

Comment:
Yeah, I'm going to leave it as-is for now for upstream compatibility.  We can change it later if we want.